index.md

```diff
@@ -237,7 +237,7 @@ curl -X POST "https://service.centrapay.com/payments/api/transactions.refund" \
 | 2          | 404       | REQUEST_NOT_FOUND                   | The provided request doesn’t exist |
 | 3          | 404       | TRANSACTION_NOT_FOUND               | The provided transaction doesn’t exist |
 | 4          | 404       | MERCHANT_NOT_FOUND                  | The provided Merchant doesn’t exist |
-| 5          | 400       | INVALID_REQUEST_ID                  | requestId must be a valid UUID |
+| 5          | 400       | INVALID_REQUEST_ID                  | RequestId must be a valid UUID |
```

Comment:
shouldn't this stay camelCase?

Reply:
I think all the variables should probably be camelCase, I was just updating it to match the rest of the docs, if you look each of the error code descriptions start with a capital letter including the variables. I can update them all to be camelCase if you agree, I was just trying to make it match the others.